example/views.py

```diff
@@ -74,8 +74,13 @@ class AuthorViewSet(ModelViewSet):
 
 
 class CommentViewSet(ModelViewSet):
-    queryset = Comment.objects.all()
+    queryset = Comment.objects.select_related('author', 'entry')
```

Comment:
After further investigation, the problem does not occur inside extract_relationships - Still looking